cloud/data-tiering/tier-data-object-storage.md

```diff
@@ -9,7 +9,7 @@ cloud_ui:
         - [services, :serviceID, operations, data-tiering]
 ---
 
-import ExperimentalPrivateBeta from 'versionContent/_partials/_experimental-private-beta.mdx';
+import ExperimentalPrivateBeta from 'versionContent/_partials/_early_access.mdx';
```

Comment:
General comment:
Usually we document the APIs. We don't have a API reference section here.
Where do we list data tiering related APIs?
.e.g see https://docs.timescale.com/api/latest/compression/add_compression_policy/

Reply:
I've added an xref, but we don't have a specific API section for data tiering, so I'm not sure where to point to. Do we need to add one?